cirq-google/cirq_google/json_resolver_cache.py

```diff
@@ -44,7 +44,6 @@ def _old_xmon(*args, **kwargs):
         'CouplerPulse': cirq_google.experimental.CouplerPulse,
         'GoogleNoiseProperties': cirq_google.GoogleNoiseProperties,
         'SycamoreGate': cirq_google.SycamoreGate,
-        'GateTabulation': cirq_google.GateTabulation,
```

Comment:
We cannot delete these json files.  We need to provide a way for people to deserialize this:

See
https://quantumai.google/cirq/dev/serialization#deprecating_a_serializable_value

An example where I recently did this was here:
https://github.com/quantumlib/Cirq/pull/5687